DeepSpeech.ipynb

```diff
@@ -446,7 +446,7 @@
     "    lstm_bw_cell = tf.nn.rnn_cell.BasicLSTMCell(n_cell_dim, forget_bias=1.0)\n",
     "    \n",
     "    # Split data because rnn cell needs a list of inputs for the BRNN inner loop\n",
-    "    layer_3 = tf.split(0, n_steps, layer_3)\n",
```

Comment:
>> sess.run(foo)
array([[1, 1],
       [2, 2],
       [3, 3],
       [4, 4]], dtype=int32)
>>> sess.run(tf.split(0, 2, foo))
[array([[1, 1],
       [2, 2]], dtype=int32), array([[3, 3],
       [4, 4]], dtype=int32)]
>>> sess.run(tf.unpack(tf.expand_dims(foo, 1)))
[array([[1, 1]], dtype=int32), array([[2, 2]], dtype=int32), array([[3, 3]], dtype=int32), array([[4, 4]], dtype=int32)]

That seems to confirm what I said - assuming the previous tensor has shape (x, y) and the new tensor has shape (a, x, y), we want x to be n_steps, but using expand/unpack will end up always having x as 1. I'm continuing trying to find an alternative.